specification/requirements.adoc

```diff
@@ -85,6 +85,12 @@ To fully support IDE, CoVE-IO compatible devices must implement the IDE Key
 Management (IDE_KM) protocol. IDE_KM messages are initiated by the TSM and sent
 over SPDM to the DSM.
 
+The CoVE-IO compatible devices must keep tracking the IDE key invocation field,
+which is the bit 63:0 of the IV. If the IDE key invocation field overflow
+happens, the IDE stream must enter Insecure. Before overflow, the device may
+signal event to the host to let the host software perform key refresh process
+with IDE_KM protocol.
+
```

Comment:
Thanks a lot @jyao1. I only have a few nits that I collected into https://github.com/jyao1/riscv-ap-tee-io/pull/1. Feel free to squash it into your commit if they make sense to you.

Reply:
Thanks. I have merged and squashed.